tsweb/tsweb.go

```diff
@@ -396,6 +397,37 @@ func VarzHandler(w http.ResponseWriter, r *http.Request) {
 				dump(name+"_", kv)
 			})
 			return
+		case PrometheusMetricsReflectRooter:
+			root := v.PrometheusMetricsReflectRoot()
+			rv := reflect.ValueOf(root)
+			if rv.Type().Kind() == reflect.Ptr {
+				if rv.IsNil() {
+					return
+				}
+				rv = rv.Elem()
+			}
+			if rv.Type().Kind() != reflect.Struct {
+				fmt.Fprintf(w, "# skipping expvar %q; unknown root type\n", name)
+				return
+			}
+			foreachExportedStructField(rv, func(fieldOrJSONName, promTyp string, rv reflect.Value) {
+				mname := name + "_" + fieldOrJSONName
+				switch rv.Kind() {
+				case reflect.Int, reflect.Int64:
```

Comment:
Add uint, uint64, and 32 bit integer variants. Someone will use them soon enough, and it's easy enough to get them right now.